protocol/xmrtaker/swap_state_test.go

```diff
@@ -345,9 +345,19 @@ func TestSwapState_NotifyClaimed(t *testing.T) {
 	t.Logf("transferred %d pico XMR (fees %d) to account %s", tResp.Amount, tResp.Fee, xmrAddr)
 	require.Equal(t, uint64(amt), tResp.Amount)
 
+	transfer, err := maker.WaitForTransReceipt(&monero.WaitForReceiptRequest{
+		Ctx:              s.ctx,
+		TxID:             tResp.TxHash,
+		DestAddr:         xmrAddr,
+		NumConfirmations: monero.MinSpendConfirmations,
+	})
+	require.NoError(t, err)
+	t.Logf("Transfer mined at block=%d with %d confirmations", transfer.Height, transfer.Confirmations)
+
 	// send notification that monero was locked
 	lmsg := &message.NotifyXMRLock{
 		Address: string(xmrAddr),
+		TxID:    transfer.TxID,
```

Comment:
Bob isn't currently using the TxID at the current time, but having the value might help with troubleshooting if nothing else.